packages/components/src/Brand/Brand.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { HTMLAttributes } from "react"
 import { assetUrl } from "@kaizen/hosted-assets"
 import { OverrideClassName } from "~types/OverrideClassName"
+import collectiveIntelligenceUrl from "./BrandCollectiveIntelligenceLogo.svg"
```

Comment:
What made you opt for using the SVG directly over making it a React component?

Reply:
Hmm... didn't really think about it. Just tried not changing the render structure too much 😬 I did a quick search for the usage of `Brand` and `classNameOverride` was being put on some of them, so wanted to avoid potentially breaking those.

Also attributes for `picture` may differ from `svg`, so the spread props will be in a funny place.

I don't mind changing it if you'd prefer? Would certainly remove the need for all the extra SVG config.